### ISSUE TITLE
Tradebot crashes : minimum is not defined

### ISSUE THREAD

**Issue by glegarand** [askmike/gekko#2160]
**Note: this is the technical bug tracker, please use other platforms for getting support and starting a (non technical) discussion. See the [getting help page](https://gekko.wizb.it/docs/introduction/getting-help.html) for details.**

**I'm submitting a ...**
[ X] bug report
[ ] question about the decisions made in the repository

**Action taken** (what you did)
I'm trading using Gekko UI on Kraken with the EUR/ZEC pair and it used to run fine, but the bot crashed after almost 2 months. I changed my strategy and the tradebot constantly crashes.
I tried to change the minimum amounts on /utils/genMarketFiles/update-kraken.js but it didn't change anything.

I have the same problem on the ETH/EOS pair on Kraken. Some other bots run on Binance and I have no problem. I use the develop branch and I'm up to date.

**Expected result** (what you hoped would happen)
Trade if the minimum amount is available, otherwise do nothing.

**Actual result** (unexpected outcome)
The tradebot crashes and disappear from the UI.

**Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, etc)
EUR/ZEC

```2018-05-09 17:53:50 (DEBUG):	[kraken.js] entering "setBalance" callback after kraken-api call, data: { error: [],
  result: 
   { ZUSD: '155.0091',
     ZEUR: '137.7853',
     XXBT: '0.0884483050',
     XXRP: '0.00000071',
     XLTC: '0.0000000000',
     XETH: '0.8195571400',
     XZEC: '0.0229425800',
     XXMR: '0.0000000000',
     EOS: '0.0000002600',
     BCH: '0.0000000000' } }
2018-05-09 17:53:50 (INFO):	trading at kraken ACTIVE
2018-05-09 17:53:50 (INFO):	kraken trading fee will be: 0.16%
2018-05-09 17:53:50 (INFO):	kraken portfolio:
2018-05-09 17:53:50 (INFO):		 EUR: 137.785300000000
2018-05-09 17:53:50 (INFO):		 ZEC: 0.022942580000
2018-05-09 17:53:50 (INFO):	Setting up:
2018-05-09 17:53:50 (INFO):		 Performance Analyzer
2018-05-09 17:53:50 (INFO):		 Analyzes performances of trades
2018-05-09 17:53:50 (INFO):	

2018-05-09 18:25:52 (INFO):	Trader Received advice to go short. Selling  ZEC
2018-05-09 18:25:52 (DEBUG):	portfolioManager : newTrade() : creating a new Trade class to  SELL ZEC / EUR
2018-05-09 18:25:52 (DEBUG):	created new Trade class to SELL ZEC/EUR
2018-05-09 18:25:53 (DEBUG):	[kraken.js] entering "setBalance" callback after kraken-api call, data: { error: [],
  result: 
   { ZUSD: '155.0091',
     ZEUR: '137.7853',
     XXBT: '0.0884483050',
     XXRP: '0.00000071',
     XLTC: '0.0000000000',
     XETH: '0.8195571400',
     XZEC: '0.0229425800',
     XXMR: '0.0000000000',
     EOS: '0.0000002600',
     BCH: '0.0000000000' } }
2018-05-09 18:25:53 (ERROR):	[kraken.js] (getPortfolio) returned an irrecoverable error: minimum is not defined```

ETH/EOS
```2018-05-09 18:23:50 (INFO):	Trader Received advice to go long. Buying  EOS
2018-05-09 18:23:50 (DEBUG):	portfolioManager : newTrade() : creating a new Trade class to  BUY EOS / ETH
2018-05-09 18:23:50 (DEBUG):	created new Trade class to BUY EOS/ETH
2018-05-09 18:23:52 (DEBUG):	[kraken.js] entering "setBalance" callback after kraken-api call, data: { error: [],
  result: 
   { ZUSD: '155.0091',
     ZEUR: '137.7853',
     XXBT: '0.0884483050',
     XXRP: '0.00000071',
     XLTC: '0.0000000000',
     XETH: '0.8195571400',
     XZEC: '0.0229425800',
     XXMR: '0.0000000000',
     EOS: '0.0000002600',
     BCH: '0.0000000000' } }
2018-05-09 18:23:52 (ERROR):	[kraken.js] (getPortfolio) returned an irrecoverable error: minimum is not defined```

**Comment by H256** [comment]
Hi, maybe this workaround helps you: https://github.com/askmike/gekko/issues/2083
There's an open PR for this, but it has not been merged yet.

**Comment by glegarand** [comment]
I updated the plugins/trader/trade.js as mentionned, but now I can't see my trades in the UI.
I have a buy ETH/EOS order recorded on Kraken this morning, but no green dot nor stats on the UI.

**Comment by H256** [comment]
@glegarand Unfortunately this seems to be another problem that could be fixed with this PR https://github.com/askmike/gekko/pull/2181  , though I have not tried it out, since the bot trades fine and I'm analysing the logs instead looking at the ui.

**Comment by askmike** [comment]
Note that there are some bugs with the trader which I am currently improving, see #2118. But yes, @H256 is correct that this bug is fixed with #2181. Please download the latest develop branch from git to get the fix.
